augur/io/file.py

```diff
@@ -2,9 +2,23 @@
 from contextlib import contextmanager
 from io import IOBase
 from textwrap import dedent
-from xopen import PipedCompressionReader, PipedCompressionWriter, xopen
+
+from xopen import xopen
+
 from augur.errors import AugurError
 
+# Workaround to maintain compatibility with both xopen v1 and v2
+# In some time, we can drop support for xopen v1
+# by removing the try-except block and using
+# _PipedCompressionProgram directly
```

Comment:
Even though it's just a dependency compatibility, this could still go in [DEPRECATED.md](https://github.com/nextstrain/augur/blob/-/DEPRECATED.md)

Reply:
Done